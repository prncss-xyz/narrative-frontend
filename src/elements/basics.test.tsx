import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import {
  ActionBox,
  Box,
  Flex,
  H1,
  H3,
  Img,
  OverlayActionBox,
} from "./basics";

describe("basics", () => {
  describe("Img", () => {
    const { container } = render(<Img />);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("Box", () => {
    const { container } = render(<Box />);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("Flex", () => {
    const { container } = render(<Flex />);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("H1", () => {
    const { container } = render(<H1>H1</H1>);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("H2", () => {
    const { container } = render(<H3>H2</H3>);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("H3", () => {
    const { container } = render(<H3>h3</H3>);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("ActionBox", () => {
    const { container } = render(<ActionBox>h3</ActionBox>);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
  describe("OverlayActionBox", () => {
    const { container } = render(<OverlayActionBox>h3</OverlayActionBox>);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
});
