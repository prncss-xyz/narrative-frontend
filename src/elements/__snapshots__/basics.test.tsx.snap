// Vitest Snapshot v1

exports[`basics > ActionBox > should match snapshot 1`] = `<div />`;

exports[`basics > Box > should match snapshot 1`] = `<div />`;

exports[`basics > Flex > should match snapshot 1`] = `<div />`;

exports[`basics > H1 > should match snapshot 1`] = `<div />`;

exports[`basics > H2 > should match snapshot 1`] = `<div />`;

exports[`basics > H3 > should match snapshot 1`] = `<div />`;

exports[`basics > Img > should match snapshot 1`] = `
<div>
  <img
    class="css-174dvwn-Img edgsz5f4"
  />
</div>
`;

exports[`basics > OverlayActionBox > should match snapshot 1`] = `<div />`;
